packages/offix-datastore/src/storage/LocalStorage.ts

```diff
@@ -82,4 +112,12 @@ export class LocalStorage {
   public async readMetadata(storeName: any): Promise<any> {
     return this.adapter.query(storeName);
   }
+
+  private dispatchEvent(event: StoreChangeEvent) {
+    if (!this.adapter.isTransactionOpen()) {
```

Comment:
So technically it will allow multiple operations going thru transition as well. All controllable by Open and commit?

Reply:
yes